### DISCUSSION TITLE
How should the loss in input.json be set

### DISCUSSION THREAD

**Discussion by 1782552114** [deepmodeling/deepmd-kit#5041] [Q&A]
Here are my parameters, the model after training the potential function is not ideal，The system is a metal system
{
    "_comment": " model parameters",
    "model": {
        "type_map": [],
        "descriptor": {
            "type": "se_e2_a",
            "sel": "auto",
            "rcut_smth": 0.50,
            "rcut": 6.50,
            "neuron": [64, 128, 256],
            "resnet_dt": false,
            "axis_neuron": 32,
            "seed": 1
        },
        "fitting_net": {
            "neuron": [128,256,512],
            "resnet_dt": true,
            "seed": 1
        }
    },

    "learning_rate": {
        "type": "exp",
        "decay_steps": 5000,
        "start_lr": 0.001,
        "stop_lr": 1e-8
    },

    "loss": {
        "type": "ener",
        "start_pref_e": 0.02,
        "limit_pref_e": 1.0,
        "start_pref_f": 1000,
        "limit_pref_f": 1,
        "start_pref_v": 0.02,
        "limit_pref_v": 1.0
    },

    "training": {
        "training_data": {
            "systems": [],
            "batch_size": "auto",
            "auto_prob": "prob_sys_size"
        },
        "validation_data": {
            "systems": [],
            "batch_size": "auto",
            "numb_btch": 3
        },
        "numb_steps": 500000,
        "seed": 10,
        "disp_file": "lcurve.out",
        "disp_freq": 1000,
        "save_freq": 10000
    }
}

**Comment by 1782552114** [comment]
<img width="4769" height="3558" alt="training_loss_comparison" src="https://github.com/user-attachments/assets/b69aec3b-4300-46e8-9319-2ade25867369" />
